Contest_API.md

```diff
@@ -770,6 +770,65 @@ The `countdown_pause_time` may be changed to indicate the approximate delay unti
 Countdown is resumed by setting a new `start_time` and resetting
 `countdown_pause_time` to `null`.
 
+In most cases downstream clients should use `/scoreboard` endpoint to get standings. If for some reason, it's not suitable, one
+can use `scoreboard_rules` property to ensure that calculated scoreboard would be the same, or at least report potential
+diverges. Scoreboard rule object must have a `string` property named `kind` and any additional properties, describing settings. 
+CCS should use one of the objects described in the [known scoring rules](#known-scoreboard-rules) section, when possible.  
+
+#### Known scoreboard rules
+
+##### ICPC scoreboard rules
+
+`ICPC` kind is set of scoreboard calculation rules, where each problem is either solved or not, teams
+are sorted based on number of solved problems, and penalty time as a tiebreaker. Can be used only with `pass-fail` 
+`scoreboard_type`.
```

Comment:
There should be a reference here to the CCS SR (or the other way around, or both).

Also, (but that's kinda the same thing), an update of the [Contest API section](https://ccs-specs.icpc.io/draft/ccs_system_requirements#contest-api) of the CCS SR is needed.

Reply:
We discussed thus, but adding it here...

The fact that `ICPC` can only be used with `pass-fail` (and that I think that will be true for many other kinds) is an argument for moving `scoreboard_type` into `scoreboard_rules`.